fuel-tx/src/transaction/types/create.rs

```diff
@@ -183,16 +168,12 @@ impl Chargeable for Create {
             ..
         } = self;
 
-        let contract: Option<Contract> = witnesses
+        let contract_len = witnesses
             .get(*bytecode_witness_index as usize)
-            .map(|c| c.as_ref().into());
+            .map(|c| c.as_ref().len())
+            .unwrap_or_default();
```

Comment:
```suggestion
            .unwrap_or(0);
```
Helps with readability.